torch/csrc/autograd/custom_function.h

```diff
@@ -95,6 +95,8 @@ struct TORCH_API AutogradContext {
   std::vector<torch::autograd::SavedVariable> saved_variables_;
   variable_list to_save_;
 
+  // The CppNode in the autograd graph that owns this AutogradContext. We need a
+  // weak_ptr to avoid a refcycle.
```

Comment:
I think this comment can be even better by explaining why `grad_fn_` will be live when you want to use it.